lizard_efcis/models.py

```diff
@@ -139,6 +138,26 @@ class ParameterGroep(models.Model):
     def __unicode__(self):
         return self.code
 
+    @property
+    def parametergroep_dict(self):
+        return {
+            'id': self.id,
+            'code': self.code,
+            'children': self.children
+        }
+
+    @property
+    def children(self):
```

Comment:
Alternative: een classmethod die in één query alles teruggeeft. Nu heb je per node een query, dus zo'n 30 queries ofzo.

Beter in één keer alles ophalen.

.... behalve dat dit niet vaak opgeroepen zal worden en het dus niets uitmaakt, dus laat het maar zo :-)